README.md

```diff
@@ -1 +1,219 @@
-# todo-list-app
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./YonasL.webp" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Good job working on the to-do list 🥇, however, make sure you fix the link to your logo because it is broken to make your README more professional 👍 

![Screenshot 2023-08-17 171558](https://github.com/Yonasle/todo-list-app/assets/57408419/5bcec8a4-4dfa-43e1-a086-981b85a324eb)